using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -613,36 +861,46 @@ Here are some examples in Objective-C and Swift:
 **Objective-C**
 
 ```objectivec
-[ACPTarget clearPrefetchCache];
+[ACPTarget resetExperience];
 ```
 
 **Swift**
 
 ```swift
-ACPTarget.clearPrefetchCache()
+ACPTarget.resetExperience()
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### clearPrefetchCache
+
+#### Syntax
+
+```javascript
+resetExperience()
+```
+
+#### Example
 
 **JavaScript**
 
 ```javascript
-ACPTarget.clearPrefetchCache();
+ACPTarget.resetExperience();
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Retrieve Location Content requests
+## retrieveLocationContent
+
+This API to sends a batch request to the configured Target server for multiple mbox locations. 
+
+The main difference with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#loadrequests) API is in usage with prefetch APIs. For a prefetched mbox, unlike `loadRequests` API, invoking this API does not send a display notification to the configured Target server. If you are not prefetching mbox content, then there is no difference with `loadRequests` API for sending batch request to Target.
```

Comment:
- In the 1st sentence, it looks like there is an extra space between the closing bracket for the URL and "API". If yes, please remove. 
- Please reword the 3rd sentence as follows:
"If you do not have an existing prefetch content call, and you send a batch request to Target, there is no difference in behaviors of the `retrieveLocationContent` and `loadRequests` APIs."